src/routes/templates/builder/[slug]/+page.ts

```diff
@@ -0,0 +1,11 @@
+/** @type {import('./$types').PageLoad} */
+export async function load({ params }) {
+    let response = await fetch(`https://test.axelzublena.com/api/templates/${params.slug}`, {
```

Comment:
Is there a reason for the response and json variables to be mutable? Otherwise they can be declared with const